vendors/info.txt

```diff
@@ -3,9 +3,9 @@ sentry (0.4.9) (Qt IFW v1.15.0)
 Comes from release v4.0.0 of eve-launcher-vendor-libraries (https://github.com/ccpgames/eve-launcher-vendor-libraries/releases/tag/v4.0.0)
 Comes from TeamCity build tagged v4.0.0 (https://teamcity.evetech.net/buildConfiguration/Applications_EveLauncher_VendorLibraries_Packagers_SentryNativeMt?mode=builds)
 
-pdm-proto (2.0.7) (Qt IFW v1.15.1)
+pdm-proto (2.1.2) (Qt IFW v1.15.1)
```

Comment:
The Qt IFW number in the parenthesis is to tell us in which version of our Qt IFW we updated to the currently used library.
Since we're making a change to the framework, we need to increase the Qt IFW number.
You do this by updating line 10 in `installerfw.pri`:
```
CCP_FRAMEWORK_STRING = 1.15.1
```
So you need to update that file, and then update the comment here to use the new number